lib/diagnostic.js

```diff
@@ -13,10 +13,10 @@ const min = function (previous, current) {
 // Write a callback for reduce that produces
 // the sum of the values in the array
 // this callback will be passed by the tests to the reduce method, e.g.
-// `let total = array.reduce(sum);`
-const sum = function (previous, current) {
-
-}
+// let total = array.reduce(sum)
+const sum = .reduce(function (previous, current) {
```

Comment:
Yup, but you actually didn't need to make this a reduce function. They were simply asking for the function that you would pass to reduce.